packages/contentful--create-contentful-app/lib/index.js

```diff
@@ -94,30 +91,30 @@ async function initProject() {
   }
 }
 
-
 (async function main() {
-
-  switch (command) {
-    case 'init':
-      await initProject();
-      break;
-
-    case 'create-definition':
+  program
+    .command('init', { isDefault: true, hidden: true })
+    .description('Bootstrap your app inside a new folder ‘app-name’')
+    .argument('[app-name]', 'App name', 'my-app')
+    .option('--npm', 'Use NPM')
+    .option('--yarn', 'Use Yarn')
+    .option('--javascript, -js')
+    .option('--typescript, -ts')
+    .action(initProject);
+
+  program
+    .command('create-definition')
+    .description(
+      'Creates an app definition for your app in a Contentful organization of your choice.'
+    )
+    .action(async () => {
       await createAppDefinition.interactive();
-      break;
+    });
 
-    case 'help':
-      printHelpText(mainCommand, localCommand);
-      break;
+  program
+    .name(chalk.cyan('@contentful/create-contentful-app'))
```

Comment:
```suggestion
    .name(chalk.cyan('create-contentful-app'))
```
Should we get rid of the scope here as well @andipaetzold ?

Reply:
Yes. I wanted to create a ticket to have a proper cleanup (including docs), so we are more consistent.